### PR TITLE
Fix responsive search bar

### PR DESCRIPTION
**Pull request by DiegoCoronaBarragan**
# Description

I use styles to adjust the search bar in mobile view.

## Screenshots

image

### CODE REVIEW

File: app/assets/stylesheets/pages/_main.scss [hunk 1]
```diff
@@ -755,3 +749,53 @@
...
+        height: 80px;
+        margin-left: 40%;
+        
+        a {
+          margin-bottom: 1rem;
+        }
+      }
+    }
+  }
+}
```
Comment (juperameza) [inline app/assets/stylesheets/pages/_main.scss:801]: New line at the end of this file

**Review by juperameza — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/pages/_main.scss (hunk 1 above)
Comment (DiegoCoronaBarragan) [reply app/assets/stylesheets/pages/_main.scss]: Fixed :D

**Review by JorgeRodz — APPROVED** [review]
LGTM!

**Review by YilKanda — APPROVED** [review]
LGTM!

**Pull request merged**